openy.profile

```diff
@@ -800,3 +804,22 @@ function openy_enable_search_api_solr_legacy() {
 
   return FALSE;
 }
+
+/**
+ * The profile name should exist in the 'core.extension.module' list.
+ *
+ * @see https://git.drupalcode.org/project/drupal/-/commit/bd13fb1a9091996e526110851c73137a01e78cb3#a96e18f6425fb567a1083b3c6f8afcfaa94b8522_133_134
+ */
+function openy_register_profile(): void {
+  $config_factory = Drupal::configFactory();
+  $extension_config = $config_factory->getEditable('core.extension');
+  // Set the default and admin theme.
+  $config_factory
```

Comment:
I've added these configurations here because they are in the profile.install, they didn't work